MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -727,7 +719,8 @@
       "Id": "UpgradeRoyalWallet",
       "RequiredItems": [
         "AreaMoonAccess",
-        "UpgradeGiantWallet"
+        "UpgradeGiantWallet",
+        "UpgradeAdultWallet"
```

Comment:
What were these here for again? Was it to prevent weird issues with item importance? If so, we might not need these anymore since anything requiring AreaMoonAccess is now immediately discarded when it comes to item importance.

Reply:
This fixes wallets appearing on bank 3 and All-Night Mask Purchase. Apparently simply having Removed By Royal Wallet requiring only Giant's Wallet wasn't enough to completely fix the issue.

Reply:
Ok, hopefully it fixes it this time.